src/pages/TableDetail.svelte

```diff
@@ -48,3 +49,5 @@
 
   <SchemaViewer app={params.app} nodes={data.schema} />
 {/await}
+
+<Footer />
```

Comment:
Instead of adding the `Footer` to every page, you could instead add it once to the `src/App.svelte` file.